docs/admin/kubeadm-init.md

```diff
@@ -0,0 +1,425 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm init
+notitle: true
+---
+{% capture overview %}
+## Kubeadm init {#cmd-init}
+{% endcapture %}
+
+{% capture body %}
+{% include_relative _kubeadm/kubeadm_init.md %}
+
+### Init workflow {#init-workflow}
+`kubeadm init` bootstraps a Kubernetes master node by executing the
+following steps:
+
+1. kubeadm runs a series of pre-flight checks to validate the system state
+   before making changes. Some checks only trigger warnings, others are
```

Comment:
You could avoid the whole concern about capitalization in this list by using a style like this:

1. Run a series of pre-flight checks ...

2. Generate the token ...

3. Generate a self-signed CA ...